### PR TITLE
Fix bug in React 16.5.0

### PR DESCRIPTION
**Pull request by razius**
- https://github.com/ethanselzer/react-cursor-position/commit/e257bdae1931c47a0a6d2fbff4e46205c1b30c83
- https://github.com/ethanselzer/react-cursor-position/releases/tag/v3.0.2

### CODE REVIEW

**Comment by razius** [comment]
This seems to fail without the version bump as well. Any clue?

**Comment by ethanselzer** [comment]
@razius - Thanks for identifying this issue and opening a PR! I think the current version of react-image-magnify is incompatible with react-cursor-position 3.x. Try installing react-cursor-position@2.5.1. See this [Code Sandbox](https://codesandbox.io/s/7j2zvj69lq), demonstrating react-cursor-position@2.5.1 working with React 16.5.0.

**Comment by razius** [comment]
I didn't have any problems running it and our fix was to basically update the version in the `package-lock.json`.

See: https://codesandbox.io/s/m4x5nk1k38

File: package.json [hunk 1]
```diff
@@ -99,7 +99,7 @@
     "fast-deep-equal": "1.0.0",
     "object-assign": "4.1.1",
     "prop-types": "15.6.0",
-    "react-cursor-position": "2.4.1",
+    "react-cursor-position": "3.0.2",
```
Comment (ethanselzer) [inline package.json]: The 3.x version of react-cursor-position is not yet compatible with react-image-magnify. Please install react-cursor-position 2.5.1 by running the command `yarn add react-cursor-position@2.5.1`. Thanks!

**Comment by razius** [comment]
Updated. Didn't see this commit https://github.com/ethanselzer/react-cursor-position/commit/e257bdae1931c47a0a6d2fbff4e46205c1b30c83 and I thought it's only fixed in `3.0.2`.

**Comment by ethanselzer** [comment]
@razius - Thanks again for raising my awareness to the React 16.5 issue for opening this PR. It turns out there were a few dependencies that needed to be upgraded. I am closing your PR in favor of #48. Sorry about that.

**Comment by razius** [comment]
No problem, glad to see it fixed.

**Pull request closed without merging**